flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendTestBase.java

```diff
@@ -996,7 +1004,7 @@ public void testKryoRegisteringRestoreResilienceWithDefaultSerializer() throws E
      *
      * @throws Exception expects {@link ExpectedKryoTestException} to be thrown.
      */
-    @Test
+    @TestTemplate
     public void testKryoRegisteringRestoreResilienceWithRegisteredSerializer() throws Exception {
```

Comment:
```suggestion
    void testKryoRegisteringRestoreResilienceWithRegisteredSerializer() throws Exception {
```
could be package private